src/components/MergeArticleList.tsx

```diff
@@ -7,10 +7,20 @@ const Container = styled.div`
 `
 
 const reads: Array<CardListItem> = [
+   {
+    title: "Technical Aspects of Ethereum Merge",
+    description: "Alchemy",
+    link: "https://www.alchemy.com/overviews/the-ethereum-merge",
+  },
   {
-    title: "Ethmerge",
-    description: "Ethmerge",
-    link: "https://ethmerge.com/",
+    title: "How to Migrate from Ropsten to Goerli",
+    description: "Alchemy",
+    link: "https://www.alchemy.com/overviews/migrate-from-ropsten-to-goerli",
+  },
+  {
+    title: "The Merge is Coming",
```

Comment:
Can we pick one of these links? I think we should use this one as it seems to be the parent page of the 'Technical Aspects of Ethereum Merge' page.

Reply:
I'd say this one offers new content, i.e. testnet timelines but probably isn't needed here - this could be included in the "Dapp and smart contract developers" section given most users won't care about testnets.

Reply:
Hey @minimalsm thanks for the comments, appreciate the speedy review! 

Completely see your point - would you mind if we kept the Migrate from Ropsten + Goerli article plus one of the other pages as it's more general? I recommend the "The Merge is Coming" page as the second article but I'm open. Thanks again!